server/server.py

```
@@ -266,6 +266,37 @@ async def handle_get_scores(request: web.Request) -> web.Response:
+    if target:
+        ws = _players.get(target)
+        candidates = [(target, ws)] if ws else []
+    else:
+        candidates = list(_players.items())
+
+    for name, ws in candidates:
+        try:
+            await ws.close(code=1001, reason="Server requested close")
+            closed.append(name)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)